src/main/java/seedu/duke/Person.java

```diff
@@ -0,0 +1,102 @@
+package seedu.duke;
+
+import java.util.ArrayList;
+
+public class Person {
+    private String name;
+    private int age;
+    private double height; //Height in meter
+    private double weight; //Wight in Kg
+    private String gender;
+    private double weightGoal;
+    private ArrayList<DailyFoodRecord> personalFoodRecord;
+
+    public Person(String name) {
+        this.name = name;
+    }
+
+    public Person(String name, int age, String gender, double height, double weight, double weightGoal){
+        this.name = name;
+        this.age = age;
+        this.gender = gender;
+        this.height = height;
+        this.weight = weight;
+        this.weightGoal = weightGoal;
+        this.personalFoodRecord = new ArrayList<DailyFoodRecord>();
+    }
+
```

Comment:
Similar to Profile Class, understand why you would want to change the name to "Person" instead but personally I feel that it's not required.